mailpoet/lib/Util/License/Features/CapabilitiesManager.php

```diff
@@ -56,7 +56,7 @@ private function isMailpoetLogoInEmailsRequired(): bool {
       return false;
     }
 
-    return (isset($this->tier) && $this->tier < self::MIN_TIER_LOGO_NOT_REQUIRED) || (isset($mailpoetLogoInEmails) && (bool)$mailpoetLogoInEmails === true);
+    return (isset($this->tier) && $this->tier < self::MIN_TIER_LOGO_NOT_REQUIRED);
```

Comment:
If `tier` is not set and `mailpoetLogoInEmails` is `true`, this will return false. Maybe a way to fix this would be:

```suggestion
if (isset($this->tier)) {
  return $this->tier < self::MIN_TIER_LOGO_NOT_REQUIRED;
}

return true;
```

Reply:
Sorry for that @alex-mpoet. It should be good now. 
I guess the difficulty here is that this capability is reversed, it removes a feature when it is true instead of giving it.
Anyways, I have modified the condition and added tests.